blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/AbstractBitbucketScmContentProvider.java

```diff
@@ -133,13 +134,15 @@ public GitContent getContent() {
     }
 
     @SuppressWarnings("unchecked")
-    @Override
-    public boolean support(@Nonnull Item item) {
+    @CheckForNull
```

Comment:
it's a rule for findbugs apparently - to ensure nulls are defended against where it is used I guess.